listenbrainz/webserver/views/pinned_recording_api.py

```diff
@@ -63,12 +63,12 @@ def pin_recording_for_user():
         log_raise_400("Invalid JSON document submitted: %s" % str(e).replace("\n ", ":").replace("\n", " "), data)
 
     try:
-        db_pinned_rec.pin(recording_to_pin)
+        recording_to_pin_with_id = db_pinned_rec.pin(recording_to_pin)
     except Exception as e:
         current_app.logger.error("Error while inserting pinned track record: {}".format(e))
         raise APIInternalServerError("Something went wrong. Please try again.")
 
-    return jsonify({"status": "ok"})
+    return jsonify({"status": "ok", "data": _pinned_recording_to_api(recording_to_pin_with_id)})
```

Comment:
Returning status: ok with data looks a bit weird!  On the top of my head, I don't think we do this elsewhere. So let's just send data and remove the status field?

Reply:
good question. We can remove it.

Reply:
@amCap1712 in this case are you thinking of `{"data": the data}`, or `return jsonify(_pinned_recording.....)`? One thing I don't like about the second option is that it makes it difficult to modify this response in the future. Keeping in mind the GET method for this returns a dictionary with a key `"pinned_recordings"`... maybe returning `"pinned_recording": whatever` (singular) is a good way of being consistent?

Reply:
"pinned_recording": whatever sounds good.